README.adoc

```diff
@@ -59,28 +78,87 @@ module "backup" {
     iam_role_arn = module.iam_assumable_role_velero.iam_role_arn
   }
 
-  dependency_ids = {
-    kube-prometheus-stack = module.kube-prometheus-stack.id,
-    argocd                = module.argocd_bootstrap.id # TODO Can cause issues on the first cluster bootstrap. If it's the case, deploy after the last argocd.
-  }
+  # Triggers an alert if too many backups are partially failed (ratio)
+  alert_partially_failed_ratio = 0.25
+  # Triggers an alert if too many backups are failed (ratio)
+  alert_failed_ratio = 0.1
+  # Triggers an alert if latest backup is older than this, in seconds
+  alert_backup_timeout = 60 * 60 * 24
+
 }
 
 ----
 
-== Technical Reference
+Once the Velero controller is deployed on the server, it is also possible to perform manual backup from the `velero` client:
```

Comment:
```suggestion
Once the Velero controller is deployed on the server, it is also possible to perform a manual backup using the `velero` client:
```

Reply:
Another question: have you documented somewhere else how this client authenticates to the `velero` controller inside the cluster?